### PR TITLE
Add getTableNameById method

### PR DESCRIPTION
**Pull request by viktor-liablin**
Add getTableNameById method

### CODE REVIEW

File: backendless.d.ts [hunk 1]
```diff
@@ -269,6 +269,8 @@ declare module Backendless {
 
         function describe(model: string | Object | Function): Promise<Object>;
 
+        function getTableNameById(model: string): Promise<Object>;
```
Comment (stanislav-grin) [inline backendless.d.ts:272]: why model? Should be `tableId`

**Review by stanislav-grin — CHANGES_REQUESTED** [review]

File: backendless.d.ts (hunk 1 above)
Comment (viktor-liablin) [reply backendless.d.ts]: done

File: backendless.d.ts [hunk 2]
```diff
@@ -269,6 +269,8 @@ declare module Backendless {
 
         function describe(model: string | Object | Function): Promise<Object>;
 
+        function getTableNameById(tableId: string): Promise<Object>;
```
Comment (Valodya) [inline backendless.d.ts:272]: does it return an `Object` or `String`?

Comment (viktor-liablin) [reply backendless.d.ts]: done

**Pull request merged**